spec/AuthenticationAdapters.spec.js

```diff
@@ -1707,6 +1707,24 @@ describe('Apple Game Center Auth adapter', () => {
       expect(e.message).toBe('Apple Game Center - invalid publicKeyUrl: invalid.com');
     }
   });
+
+  it('validateAuthData invalid public key http url', async () => {
+    const authData = {
+      id: 'G:1965586982',
+      publicKeyUrl: 'http://static.gc.apple.com/public-key/gc-prod-4.cer',
+      timestamp: 1565257031287,
+      signature: '1234',
+      salt: 'DzqqrQ==',
+      bundleId: 'cloud.xtralife.gamecenterauth',
+    };
+
+    try {
+      await gcenter.validateAuthData(authData);
```

Comment:
This looks odd because it's a promise that throws instead of being rejected, but maybe the adapter works that way?

Anyway, if it throws, could you simplify this to `expect(...).toThrow(...)`?

Reply:
I will check to see if expect covers it.

if you look at the original codecov this type of test was missing. The server had code for it, but never checked it. Apple only allows tls connections.

Reply:
I can't get the expect.toThrow to work. I tried a number of different ways:

```javascript
await expectAsync(gcenter.validateAuthData(authData))
      .toBeRejectedWith("Apple Game Center - invalid publicKeyUrl: http://static.gc.apple.com/public-key/gc-prod-4.cer");

await expectAsync(gcenter.validateAuthData(authData))
      .toThrow("Apple Game Center - invalid publicKeyUrl: http://static.gc.apple.com/public-key/gc-prod-4.cer");
```

If you know how to do it, let me know. Feel free to edit the test directly if you wish.

Reply:
Since this throws, it may not need `await`. Did you try 
```
expect(gcenter.validateAuthData(authData)).toThrow("Apple Game Center - invalid publicKeyUrl: http://static.gc.apple.com/public-key/gc-prod-4.cer");
```

Reply:
I tried that along with other variations, it didn’t work for me.